### ISSUE TITLE
[Bug] Async generators

### ISSUE THREAD

**Issue by dufia** [lukehaas/RunJS#119]
This snipped should log numbers from 1 to 5:

```
async function* generateSequence(start, end) {

  for (let i = start; i <= end; i++) {

    // yay, can use await!
    await new Promise(resolve => setTimeout(resolve, 1000));

    yield i;
  }

}

(async () => {

  let generator = generateSequence(1, 5);
  for await (let value of generator) {
    console.log(value); // 1, then 2, then 3, then 4, then 5
  }

})();
```

You can test it by pasting it into Chrome's console or node.

**Comment by lukehaas** [comment]
@dufia thanks for raising this. I will investigate.

**Comment by yonifra** [comment]
also, multiple `.next()` invocations only show the first one

for example:
```function* generator() {
  yield 'hey'
  console.log('after hey')
  yield 'wow'
  console.log('after wow')
  return 'Done!'
}

generator().next()
generator().next()
generator().next()```
will output `{ value: 'hey', done: false }` where in Chrome console it will print `hey`, `wow` and `Done!`

**Comment by lukehaas** [comment]
@dufia @yonifra this issue is now resolved in RunJS version 1.10.0
